docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -323,7 +323,141 @@ Use this to produce a PDF of the band structure:
            ...: bands = load_node(<PK>)  # PK of the `band_structure` node
            ...: bands.show_mpl(y_origin=fermi_energy, plot_zero_axis=True)
 
+Querying the database
+---------------------
+
+As you will use AiiDA to run your calculations, the database that stores all the data and the provenance will quickly grow to be very large.
+To help you find the needle that you might be looking for in this big haystack, we need an efficient search tool.
+AiiDA provides a tool to do exactly this: the ``QueryBuilder``.
+The ``QueryBuilder`` acts as the gatekeeper to your database, to whom you can ask questions about the contents of your database (also referred to as queries), by specifying what are looking for.
+In the final part of the tutorial, we will show an short demo on how to use the ``QueryBuilder`` to make these queries and understand/use the results.
+
+First, we'll import a set of previously created data entries that we'll use in this section:
```

Comment:
Should we maybe explain a bit where does this DB came from and what it contains? I know its not super relevant, but I feel just saying this is like "so first we need some data that came from somewhere and has some stuff".

Reply:
I know the database contains perovskites, but I don't know its origin. Perhaps something along the lines of:

_"First, we'll import an archive of calculations performed on a group of 57 perovskite structures, using three different functionals: LDA, PBE and PBEsol."_